tickets/src/__tests__/components/content/CreateContent.test.js

```diff
@@ -194,7 +207,7 @@ describe('CreateContent', () => {
     expect(loadEvent).toHaveBeenCalledWith('abc123')
   })
 
-  it('should reset the fields when a different lock is selected', () => {
+  it('should load the corresponding event when a different lock is selected', () => {
```

Comment:
It will not update the form right away with the new data, but will trigger an event to load it from the server.